pandas/tests/series/test_analytics.py

```diff
@@ -1381,8 +1381,12 @@ def test_ptp(self):
         ser = Series(arr)
         assert np.ptp(ser) == np.ptp(arr)
 
-        # GH11163
         s = Series([3, 5, np.nan, -3, 10])
+
+        # GH21614
```

Comment:
make this another method. you need to catch any warnings for calling ``s.ptp()`` anywhere in the codebase, or just call ``np.ptp``